dashboard/app/models/course_version.rb

```diff
@@ -11,7 +11,7 @@
 #  created_at         :datetime         not null
 #  updated_at         :datetime         not null
 #  course_offering_id :integer
-#  published_state    :string(255)      default("in_development")
+#  published_state    :string(255)
```

Comment:
What remove the default?

Reply:
oh good catch, that was a merge issue. I'll fix